packages/frontend-app/src/app/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { Toast } from "./Toast";
```

Comment:
Please can you rewrite the story such that a user can click on a button and it displays the toast. This way I don't need to always refresh the entire page.